models/trello_schema.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
+
+sources:
+  - name: trello_test
+    database: trello_test
+    description: This database contains relations created by tap-trello
+    schema: analytics
```

Comment:
Schema should be: `"{{ env_var('DBT_TARGET_SCHEMA') }}"`. This way whatever we can use the schema the user sets, rather than expecting it to be analytics.

Reply:
resolved!